### PR TITLE
[MIG][14.0] :delivery dropoff_site

### PR DESCRIPTION
**Pull request by BechirAouji**
ping @flotho

### CODE REVIEW

**Comment by flotho** [comment]
superseeds https://github.com/OCA/delivery-carrier/pull/491

**Comment by flotho** [comment]
@BechirAouji , didi you checked the reusts of @rousseldenis from #491 If so ask for a review

**Comment by BechirAouji** [comment]
Ping @saxomanu
A review is needed please .

File: delivery_dropoff_site/models/stock_move.py [hunk 1]
```diff
@@ -2,17 +2,23 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import models
+from odoo import fields, models
 
 
 class StockMove(models.Model):
     _inherit = "stock.move"
 
+    final_shipping_partner_id = fields.Many2one(
```
Comment (rousseldenis) [inline delivery_dropoff_site/models/stock_move.py:11]: As moving this field, you should provide migration script.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Just one comment

File: delivery_dropoff_site/models/stock_move.py (hunk 1 above)
Comment (flotho) [reply delivery_dropoff_site/models/stock_move.py:11]: @saxomanu @BechirAouji , could you remind us why this field has been ported on the stock move without any links to the same field in the stock.picking .

File: delivery_dropoff_site/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        comodel_name="res.partner",
+        string="Final Recipient",
+        help="It is the partner that will pick up the parcel " "in the dropoff site.",
+    )
+
+    def _get_new_picking_values(self):
+        res = super(StockMove, self)._get_new_picking_values()
+        res.update(
+            {
+                "final_shipping_partner_id": self.final_shipping_partner_id.id,
```
Comment (flotho) [inline delivery_dropoff_site/models/stock_move.py:21]: don't sure to understand why procurement group has disappear https://github.com/OCA/delivery-carrier/blob/10.0/delivery_dropoff_site/models/stock_move.py#L16

could you explain what's hiding there ?

File: delivery_dropoff_site/models/procurement_group.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2018 - Today: GRAP (http://www.grap.coop)
```
Comment (flotho) [inline delivery_dropoff_site/models/procurement_group.py:1]: please change this file name to match the object name in the class

File: delivery_dropoff_site/models/stock_move.py (hunk 2 above)
Comment (flotho) [reply delivery_dropoff_site/models/stock_move.py:21]: ok, the procurement group don't exists anymore.

**Comment by rousseldenis** [comment]
@flotho procurement group still exists.

Procurement orders have been removed in v11.

**Comment by flotho** [comment]
thanks @rousseldenis , we'll have a look to this with @BechirAouji and @saxomanu

**Pull request closed without merging**